tests/antares/services/local_services/test_area.py

```diff
@@ -1306,3 +1307,76 @@ def test_read_misc_gen_local(self, local_study_w_areas):
             _write_file(file_path, expected_time_serie)
             matrix = area.get_misc_gen_matrix()
             pd.testing.assert_frame_equal(matrix, expected_time_serie)
+
+
+class TestReadThermal:
+    def create_file_with_empty_dataframe(self, file_path: str, df: pd.DataFrame):
```

Comment:
You don't use this method, you can remove it

Reply:
Ok, for the last commit